packages/sql-mapper/test/entity.test.js

```diff
@@ -369,6 +369,39 @@ test('[SQLite] throws if PK is not INTEGER', { skip: !isSQLite }, async ({ fail,
   }
 })
 
+test('[SQLite] allows to have VARCHAR PK', { skip: !isSQLite }, async ({ same, teardown }) => {
+  async function onDatabaseLoad (db, sql) {
+    await clear(db, sql)
+    teardown(() => db.dispose())
+
+    try {
+      await db.query(sql`CREATE TABLE pages (
+        id varchar(255) PRIMARY KEY,
+        title varchar(255) NOT NULL
+      );`)
+    } catch (error) {
+      console.log(error)
+    }
+  }
```

Comment:
If this is intended to work, we shouldn't catch the error, and we should do more than just log it.

Reply:
so you'd suggest removing the catch clause?

Reply:
Well what is the purpose of the `try...catch`?

Reply:
the `try clause`, a query is run on a db to create an entry with a `varchar` primary key. the `catch clause` to catch any error incase the query fails.

Reply:
Is the query ever expected to fail under normal conditions? If it's not expected to fail, then don't catch the error so that the test fails.

Reply:
no the test query is not expected to since `varchar` type is among the expected types. 

Alright, let me remove the`catch` block

Reply:
@cjihrig i have removed the entire `try-catch` block via [3bf9d9f](https://github.com/platformatic/platformatic/pull/643/commits/3bf9d9f0124a6e473017768c948b0fdbd4adeea8)